package.json

```diff
@@ -37,7 +37,8 @@
     "nobuild:unit": "npm run mocha:ts -- --timeout 60000 'test/*.ts'",
     "prettier": "prettier 'ts_src/**/*.ts' 'test/**/*.ts' --ignore-path ./.prettierignore",
     "prettierjs": "prettier 'src/**/*.js' --ignore-path ./.prettierignore",
-    "test": "npm run build && npm run format:ci && npm run lint && npm run nobuild:coverage",
+    "checks": "npm run build && npm run format:ci && npm run lint && npm run nobuild:coverage",
+    "test": "npm run build && npm run nobuild:unit && npm run nobuild:integration",
```

Comment:
```suggestion
    "test": "npm run nobuild:unit && npm run nobuild:integration",
```

Reply:
@louisinger we should use these commands in the gh action.

Reply:
@louisinger we should use these commands in the gh action.

Reply:
better to keep several steps in the GH actions, `test` is for local testing only @altafan